src/modules/display.js

```diff
@@ -0,0 +1,215 @@
+  commentsDiv.id = 'comments';
+
+  getMovieComments(id).then((response) => {
+    if (response.error) {
+      commentsDiv.innerHTML = '  <h4>Comments (0)</h4> <p> There are no comments yet.</p>';
+    } else {
+      commentsDiv.innerHTML = `<h4>Comments (${getMovieCommentCounter(response)})</h4>`;
+
+      const commentsList = document.createElement('ul');
+      commentsList.classList.add('comments-list');
+      response.forEach((comment) => {
+        commentsList.innerHTML += `<li> ${comment.username} <span>${moment(comment.creation_date).fromNow()}</span>\n <p> ${comment.comment}</p></li>`;
+      });
+      commentsDiv.appendChild(commentsList);
+    }
+  }, () => {
+    commentsDiv.innerHTML = '<p> There are no comments yet.</p>';
+  });
+  contentSide.appendChild(commentsDiv);
+};
+
+const showMovieDetails = (id) => {
+  const wrapper = document.createElement('div');
+  wrapper.classList.add('popup-wrapper');
+  wrapper.id = 'popup';
+
+  const popup = document.createElement('div');
+  popup.classList.add('popup');
+
+  const closeBtn = document.createElement('button');
+  closeBtn.type = 'button';
+  closeBtn.classList.add('close-btn');
+  closeBtn.innerHTML = '<i class="fa fa-close"></i>';
+  closeBtn.onclick = () => {
+    const popupShowing = document.getElementById('popup');
+    document.body.removeChild(popupShowing);
+  };
+  popup.appendChild(closeBtn);
+
+  const contentContainer = document.createElement('div');
+  contentContainer.classList.add('content-container');
+
+  const contentSide = document.createElement('div');
+  contentSide.classList.add('content-side');
+
+  const form = document.createElement('form');
+  form.classList.add('add-comment');
+
+  const headingText = document.createElement('h4');
+  headingText.innerText = 'Add a comment';
+  form.appendChild(headingText);
+
+  const nameInput = document.createElement('input');
+  nameInput.type = 'text';
+  nameInput.placeholder = 'Your name';
+  nameInput.name = 'name';
+  nameInput.minLength = 2;
+  nameInput.required = true;
+  form.appendChild(nameInput);
+
+  const commentInput = document.createElement('textarea');
+  commentInput.placeholder = 'Your insights';
+  commentInput.cols = 40;
+  commentInput.rows = 6;
+  commentInput.name = 'comment';
+  commentInput.minLength = '2';
+  commentInput.required = true;
+  form.appendChild(commentInput);
+
+  const submitBtn = document.createElement('button');
+  submitBtn.type = 'submit';
+  submitBtn.classList.add('add-comment');
+  submitBtn.innerText = 'comment';
+  form.appendChild(submitBtn);
+
+  form.onsubmit = (event) => {
+    event.preventDefault();
+    submitBtn.innerHTML = '<i class="fa fa-spinner"></i> Loading...';
+    addMovieComment({ item_id: id, username: nameInput.value, comment: commentInput.value }).then(
+      () => {
+        submitBtn.innerText = 'comment';
+        form.reset();
+        fetchMovieComments(id, contentSide, true);
+      },
+      () => {
+        // alert(`${error}`);
```

Comment:
- [ ] I would recommend removing the commented code to make the codebase cleaner.